docs/source/user_guide/installation/GNU-linux.rst

```diff
@@ -133,9 +131,29 @@ order to use the wrapped SUNDIALS ODE and DAE
 
    .. code:: bash
 
-	  pip install scikits.odes
+      brew install openblas
+      pybamm_install_odes
```

Comment:
Should we add `gcc` and `gfortran` here?

```suggestion
      brew install openblas gcc gfortran
      pybamm_install_odes
```

Reply:
Yes, better to add.